<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="LAPTOP-7DB4GVKK" name="com.orangehrm.qa.tests.Test_Performance_Page" tests="6" failures="0" timestamp="2022-09-07T23:04:01 IST" time="16.247" errors="1">
  <testcase name="testEmpName" time="0.000" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <skipped/>
  </testcase> <!-- testEmpName -->
  <testcase name="testJobTitle" time="0.000" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <skipped/>
  </testcase> <!-- testJobTitle -->
  <testcase name="beforeMethod" time="16.247" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.orangehrm.qa.basetest.BaseClass.beforeMethod(BaseClass.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:510)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
    </error>
  </testcase> <!-- beforeMethod -->
  <testcase name="testSubUnit" time="0.000" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <skipped/>
  </testcase> <!-- testSubUnit -->
  <testcase name="testReviewStatus" time="0.000" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <skipped/>
  </testcase> <!-- testReviewStatus -->
  <testcase name="testIncludeEmp" time="0.000" classname="com.orangehrm.qa.tests.Test_Performance_Page">
    <skipped/>
  </testcase> <!-- testIncludeEmp -->
</testsuite> <!-- com.orangehrm.qa.tests.Test_Performance_Page -->
